tests/playwright-test/playwright.expect.misc.spec.ts

```diff
@@ -230,6 +230,49 @@ test('should support toHaveClass w/ array', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(1);
 });
 
+test('should support toContainClass', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.test.ts': `
+      const { test } = pwt;
+
+      test('pass', async ({ page }) => {
+        await page.setContent(
+          '<div class="foo bar baz"><span class="alice item baz"></span><span class="bob item baz"></span></div>'
+        );
+        const locator = page.locator('div');
+        await expect(locator).toContainClass('foo');
+        await expect(locator).toContainClass('bar');
+        await expect(locator).toContainClass('baz');
+        await expect(locator).toContainClass('foo baz');
+        await expect(locator).not.toContainClass('ba');
+
+        const locatorSpan = page.locator('div span');
+        await expect(locatorSpan).toContainClass(['alice baz', 'bob']);
+        await expect(locatorSpan).not.toContainClass(['alice', 'alice']);
+      });
+
+      test('fail', async ({ page }) => {
+        await page.setContent('<div class="bar baz"></div>');
+        const locator = page.locator('div');
+        await expect(locator).toContainClass('foo', { timeout: 1000 });
+      });
+
+      test('fail', async ({ page }) => {
```

Comment:
```suggestion
      test('fail length mismatch', async ({ page }) => {
```
lets rename this, since PWT (Playwright Test) does not allow 2 tests at the same time. Thats why you had 3 tests in source but only 1 passed and 1 failed.